docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

```diff
@@ -1259,14 +1259,23 @@ suites:
 
 <p><small>| OPTIONAL | ARRAY |</small></p>
 
-Name/path and (optionally) output file path of each formatter to use. See the [Cucumber.js Formatters documentation](https://github.com/cucumber/cucumber-js/blob/main/docs/formatters.md) for more information.
+Specifies the name or path of each formatter to use, along with an optional output file path. If no output file path is specified, the output defaults to stdout.
+The name or path, as well as the optional output file path, must be enclosed in quotes. See the [Cucumber.js Formatters documentation](https://github.com/cucumber/cucumber-js/blob/main/docs/formatters.md) for more information.
+
+:::warning
+Cucumber deprecated the previous `format` configuration without quotes in version **9.6.0** and will remove it in **11.0.0** or later.
+If you're still using a `format` setting without quotes, migrate to the latest pattern.
+For more details, see the [Cucumber.js Deprecations documentation](https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md#ambiguous-colons-in-formats).
+
+The `file://my_formatter/implementation:output_file` is an example of ambiguous colon usage and is not allowed in Sauce Labs now. You should migrate to the quoted pattern.
+:::
```

Comment:
I think this level of detail makes things a little more confusing. Also, I'm not entirely sure what's written here is correct. The deprecation docs says:

> Cucumber tries to detect and handle things like Windows drives and file:// URLs on a best-effort basis, but this logic is being removed

And that suggests to me its not unquoted values that will be deprecated, its the logic to handle ambiguous colons.

Plus if the runner handles all the quoting anyways, does the user need to know all this?

Reply:
hmm.. I agree adding this specific case here is confusing. But I have no idea whether I should point out the `file://my_formatter/implementation:output_file` can't be used anymore if some users did set it before. At least this is a breaking change. Besides, I don't want to put it in the YAML example to explicitly let them know there is a `"file://"` option.

How about we just mention Cucumber deprecated the previous format and leave a link to their docs? And use the latest pattern in our YAML example.